src/main.rs

```diff
@@ -485,6 +506,28 @@ impl AsyncComponent for App {
             gtk::glib::Propagation::Stop
         });
 
+        #[cfg(target_os = "linux")]
+        let tray_img = Icon::from_png_bytes(&TRAY_IMAGE.to_vec()).expect("icon");
+        #[cfg(target_os = "windows")]
+        let tray_img = Icon::from_resource(1, None).expect("icon");
+
+        let sender_c = sender.clone();
+        let tray = TrayIconBuilder::new()
+            .with_icon(tray_img)
+            .with_tooltip("Demo System Tray")
```

Comment:
It is not a "Demo System Tray" though, right?

Reply:
change to Space Acres